### PR TITLE
:book: Initial ROADMAP plans

### PR DESCRIPTION
**Pull request by jwmatthews**

### CODE REVIEW

File: ROADMAP.md
Comment (rromannissen) [inline ROADMAP.md:68]: Typo: "facilitate"

**Review by rromannissen — CHANGES_REQUESTED** [review]
Aside from the typos, LGTM!

File: ROADMAP.md
Comment (jwmatthews) [reply ROADMAP.md:68]: Thank you

File: .trunk/configs/.markdownlint.yaml
Comment (jwmatthews) [inline .trunk/configs/.markdownlint.yaml:12]: Note this config for markdownlint is to help address the below

I want the ability to use the same header value nested in different levels
```
# v1.0
## Summary

# v2.0
## Summarty
```

We need to use 'siblings_only` for this ability.

Below is what I was seeing prior
$ trunk check ROADMAP.md               
Checking 100% [===========================================================================================================================>]  14/14  1.5s 
                                                                                                                                                          
  ISSUES  

ROADMAP.md:144:0
 144:0  low  Multiple headings with the same content  markdownlint/MD024
 148:0  low  Multiple headings with the same content  markdownlint/MD024
 161:0  low  Multiple headings with the same content  markdownlint/MD024
 165:0  low  Multiple headings with the same content  markdownlint/MD024
 172:0  low  Multiple headings with the same content  markdownlint/MD024
 176:0  low  Multiple headings with the same content  markdownlint/MD024

File: ROADMAP.md
Comment (jwmatthews) [reply ROADMAP.md]: Typo addressed

File: ROADMAP.md
Comment (pranavgaikwad) [inline ROADMAP.md:5]: Ouline/Outline

**Review by pranavgaikwad — CHANGES_REQUESTED** [review]

File: ROADMAP.md
Comment (jwmatthews) [reply ROADMAP.md]: Thank you, fixed

**Review by pranavgaikwad — APPROVED** [review]
LGTM

**Comment by jwmatthews** [comment]
Going to merge as we want to get this published today. 

Normally I'd wait for @rromannissen to update this approval formally, but we chatted in a meeting and he confirmed he's good with Roadmap, just a timezone delay to get this formally approved....hence merging.

**Pull request merged**
